### PR TITLE
WIP Introduce Spreed Me

### PR DESCRIPTION
**Pull request by enoch85**
cc @ezraholm50 @Pe46dro 

Fails in current state

### CODE REVIEW

**Comment by Pe46dro** [comment]
Some errors i found.
Line 21-22
```
wget -q $SPREEDVER_REPO -P $NCDIR/apps
tar -zxf $NCDIR/apps/$SPREED_FILE -C $NCDIR/apps
```
The script download a `zip file` and try to extract a `tar file`

Line 32
```CAT <<-SPREEDCONF > "$SPREEDCONF"```
The command must be lower case

I will make an accurated test during the weekend if i find time

File: spreedme.sh [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+apt-add-repository ppa:strukturag/spreed-webrtc
+apt-get update
+apt-get install spreed-webrtc -y
+
+# Download and install Spreed
+if [ -d $NCDIR/apps/spreedme ]; then
+echo "Spreed-webrtc exists..."
+else
+wget -q $SPREEDVER_REPO -P $NCDIR/apps
+tar -zxf $NCDIR/apps/$SPREED_FILE -C $NCDIR/apps
```
Comment (Pe46dro) [inline spreedme.sh]: The downloaded file is a zip file not a tarball

File: spreedme.sh [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+mv $NCDIR/apps/nextcloud-spreedme-master $NCDIR/apps/spreedme
+rm $NCDIR/apps/$SPREEDVER_FILE
+fi
+
+# Enable Spreedme
+if [ -d $NCDIR/apps/spreedme ]; then
+sudo -u www-data php $NCDIR/occ app:enable spreedme
+fi
+
+CAT <<-SPREEDCONF > "$SPREEDCONF"
```
Comment (Pe46dro) [inline spreedme.sh]: The `cat`command should be lower case

**Comment by enoch85** [comment]
@Pe46dro I will give you write access to this repo, use it with caution.

Please make the changes you found.

Thanks.

**Comment by ezraholm50** [comment]
Thanks, this was just a mockup with some copy and pasting from other scripts. Will help testing as soon as the new server is up.

**Comment by Pe46dro** [comment]
I will test it and i will give you a feedback

**Comment by enoch85** [comment]
@Pe46dro Thanks! :100:

**Comment by Pe46dro** [comment]
I'm unable to test it directly on VM because i have problem with VMWare/VirtualBox.

I fixed the download of the zip, at the moment we have to upddate the plugin version on the code, i think that instead of downloading the release we should clone the git repo and maybe delete the .git to remove versions. The spreedwebrtc service don't support restart/reload so i stop and start it.

If it's possible to you try to check if all work, if you find error please attach the log.

**Comment by enoch85** [comment]
Thanks @Pe46dro for your work on this! 

Maybe we can borrow some code from the snap? https://github.com/nextcloud/spreedme-snap

**Comment by MariusBluem** [comment]
...why are we not simply using the snap inside of the VM?! The VM is using 16.04 with snap-support anyway 😁 @enoch85

**Comment by Pe46dro** [comment]
I never used snap, i don't know how it works, i have to try but i'm busy.

**Comment by enoch85** [comment]
@MariusBluem Actually, that might work. Also busy now, but welcome to try it out and add it to this PR. Thanks!

**Comment by enoch85** [comment]
I'm working on a new file, will post it soon. This can be closed.

**Comment by enoch85** [comment]
Here's the new script: https://github.com/nextcloud/vm/commit/99a9ac769af16f29d424c7aa54ffe5f4f4f6df87

TODO: Fix shared secret key.

**Pull request closed without merging**